lib/ansible/module_utils/facts/system/aix.py

```diff
@@ -0,0 +1,257 @@
+# AIX specific facts
+# Copyright: (c) 2018, Christian Tremel (@flynn1973)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+import sys
+import shlex
+import os
+import platform
+import re
+import itertools
+import subprocess
+try:
+    import json
+except ImportError:
+    import simplejson as json
+
+
+from ansible.module_utils.facts.utils import get_file_content
+
+
+from ansible.module_utils.facts.collector import BaseFactCollector
+
+
+class AIXSystemFactCollector(BaseFactCollector):
+    """
+    Some handy AIX facts, more or less nice to haves
+    """
+    name = 'aix_system_facts'
+    _fact_ids = set()
+    _platform = 'AIX'
+
+    def get_lssrc_facts(self, module=None, collected_facts=None):
```

Comment:
Complement the docstring adding the parameters, might be useful (to the other functions as well)